certified-connectors/Cognito Forms/apiDefinition.swagger.json

```diff
@@ -338,37 +338,6 @@
             "x-ms-summary": "Entry Id",
             "type": "string",
             "x-ms-visibility": "important"
-          },
-          {
-            "name": "entry",
```

Comment:
This parameter isn't actually valid for this endpoint. We haven't released this action yet, so there should be no changes for users.